### PR TITLE
Add patch to remove unnecessary warning

### PR DESCRIPTION
**Pull request by isuruf**
Needed for sage where the warning is treated as an error in doctests.

Also updates to conda-build=3

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, please rerender

**Comment by isuruf** [comment]
Tests pass. Ready for review

File: recipe/remove_unnecessary_warning.diff [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+-#if 1 /* 16/III-2016 */
++#if 1 /* 01/III-2018 */
+       if (((glp_iocp *)T->parm)->flip)
+-#if 0 /* 20/I-2018 */
+-         xprintf("WARNING: LONG-STEP DUAL SIMPLEX WILL BE USED\n");
+-#else
+-         xprintf("Long-step dual simplex will be used\n");
+-#endif
++         if (T->parm->msg_lev >= GLP_MSG_ALL)
++            xprintf("Long-step dual simplex will be used\n");
```
Comment (jakirkham) [inline recipe/remove_unnecessary_warning.diff]: Should we keep `Warning: `?

Comment (jjhelmus) [reply recipe/remove_unnecessary_warning.diff]: Upstream does not retain the warning, I believe it should be removed.

Comment (isuruf) [reply recipe/remove_unnecessary_warning.diff]: This is a patch from upstream that's already applied in their development repo, so I don't want to change it.

**Comment by jjhelmus** [comment]
Thanks @isuruf

**Pull request merged**